client/public/js/stores/EventStore.js

```diff
@@ -4,15 +4,21 @@ var EventEmitter = require('events').EventEmitter,
     AppConstants = require('../constants/AppConstants');
 
 var _shepherdEvents = [];
+var _sheepEvents = [];
 var _notShepherdEvents = [];
 var _shepherdEventsIds = [];
-var _currentEvent = {name: '', location: ''};
+var _isAttendeeToNotShepherdEvents = {};
```

Comment:
Deleted. It was to cache some of the lookups, but we do not use this implementation anymore. Also, it was to avoid doing lookups in the array.